### DISCUSSION TITLE
Cannot find MPRAGEMETA.csv in ADNI

### DISCUSSION THREAD

**Discussion by ifeelgitty** [aramis-lab/clinica#1369] [Q&A] [answered]
Hi,

I'm trying to convert the ADNI dataset to the BIDS format. As mentioned in the docs, changes have been made to the clinical dataset. I can find all other required files, but I cannot find the MPRAGEMETA.csv. I have:
- Downloaded/extracted (f)MRI data with the advanced image search. This also included the Metadata.
- Downloaded/extracted all clinical csv-files/data from the ARC Collection Builder
- Downloaded/extracted all study files (seemingly also moved to the ARC collection builder)
I've also changed the names to remove the "collection" prefix and the date-suffix, and have manually checked all files.

Considering that other people seemingly only had issues after this step, I'm wondering if I might be missing something very simple?

Best, and thank you in advance!
Sönke

**Comment by AliceJoubert** [comment, marked as answer]
Hello Sönke !

The issue with the ADNI converter is indeed the new organisation of metadata files on the ARC builder. From what I am reading it seems you did right so far. The MPRAGE file is needed if you plan on converting T1 images. Do you mind contacting us on our google [group](https://groups.google.com/g/clinica-user) to address this issue if you actually need it ?

Best,

Alice

**Reply by ifeelgitty** [reply]
Moved the discussion to the clinica google group.

**Comment by tqinger** [comment]
May I ask if you have obtained the relevant documents?

**Reply by Adam-Ismaili-92** [reply]
> May I ask if you have obtained the relevant documents?

Hello @tqinger,

If not already done, please submit your request to the [support group](https://groups.google.com/g/clinica-user) so that it can be handled more efficiently.

Best regards,

Adam ISMAILI
